style.css

```diff
@@ -0,0 +1,1165 @@
+    .block-text9{
+        margin-left: 57%;
+        margin-bottom: 1%;
+        width: 38%;
+        margin-top: 5%;
+        border: 1px solid black;
+        height: 3rem;
+        padding-top: 1%;
+    }
+    
+}
+
+@media screen and (min-width: 768px) and (max-width: 1024px){
+
+    .fifth{    height: 133rem;    }
+}
+
+/* 10 - tenth container - editor's picks */
+
+.tenth h1{
+    font-size: 1.2em; font-weight: bold; padding-top: 1%;
+    display: inline-block; margin-left: 0.5%;
+}
+
+.block-text img{width: 30px;}
+
+.box10{
+    height: 105vw;
+}
+
+.box10 img{ padding-top: 5%; width: 90vw; }
+
+.box10 h2{
+    font-size: 1.2em;
+    font-weight: bold;
+    padding-top: 1%;
+}
+
+.box10 p{font-size: 0.9em;}
+
+.image-cont{
+    position: relative;
+}
+
+.tag10{
+    height: 2rem;
+    width: 22%;
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    background: white;
+    opacity: 0.8;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.tag10 h3{color: #f72614; font-weight: bold; font-size: 1.1em; padding-top: 8%;}
+
+@media (min-width: 740px){
+    .box10{height: 40vw;}
+    .box10 img{padding-top: 5%; width: 20vw;}
+}
+
+
+
+/* 11 - eleventh container - slide shows */
+
+.box11{
+
+    height: 87vw;
+    margin-top: 1%;
+ 
+}
+
+.eleventh h1{
+    font-size: 1.2em; font-weight: bold; padding-top: 1%;
+    display: inline-block; margin-left: 0.5%;
+}
+
+.box11 img{
+    width: 87vw;
+    padding-top: 1%;
+    } 
+
+
+.box11 h2{font-size: 1em; font-weight: bold; margin-top: 2%;}
+
+.image-cont11{
+    position: relative;
+    width: 90%;
+}
+
+.tag11{
+    height: 2.2rem;
+    width: 22%;
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    background: white;
+    opacity: 0.8;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.tag11 h3{color: #f72614; font-weight: bold; font-size: 1.2em;
+     padding-top: 7%; width: 40%; border: 1px solid #f72614;
+    text-align: center; margin-top: 2%; }
+
+@media (min-width: 768px){
+
+.box11{padding-left: 1.5%;
+        height: 29vw;}
+
+.box11 img{
+    width: 28vw;
+    padding-top: 2%;
+                        }  }
+
+/* 12 - twelfth container - off-topics */
+
+.box12{
+    width: 100%;
+}
+
+
+.box12-up{
+    height: 92vw;
+    padding: 2%;
+
+}
+
+.box12-up img{
+    width: 92vw; padding-top: 3%; }
+
+
+.box12-up h2{
+        font-size: 1em; font-weight: bold; padding-top: 2%;}
+
+
+.box12-down{
+    height: 30vw;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    border-bottom: 1px solid gainsboro;
+}
+
+.noborder{border: none;}
+
+.box12-down img{
+    width: 35vw; padding-top: 3%; }
+
+
+
+.box12-down h2{
+        width: 50%; padding-top: 3%; font-size: 0.9em; font-weight: bold; }
+
+
+.title-off{
+    color: #f72210;
+    font-weight: bold;
+    font-size: 1.2em;
+    margin-left: 1.6%;
+    margin-bottom: -7.7%;
+}
+
+
+@media screen and (min-width: 768px){
```

Comment:
Confused by all of these media queries, don't you already have this media query defined in line 704.

Please correct, have only on media query definition for each screen size. It's confusing otherwise. That being said, it happens!

Reply:
I have been dividing the different parts of the website in sections, so I did a separated media query for each section just because I thought it was better organized like that, I could have also put everything in the same media query and divided that media query into different sections as well. But for me, this looks more structured, anyways, correct me if I am wrong and whether this approach is not acceptable.

Reply:
fixed in the last commit

Reply:
Hi, it's fine this way, was only wondering. 
We all have different ways of organizing code. 😄